src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue: rgba(0, 172, 220, 1);
+}
+
+body {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  display: flex;
+  text-transform: uppercase;
+  list-style: none;
+  text-decoration: none;
+}
+
+.nav_item {
+  margin-right: 20px;
+  height: 100%;
```

Comment:
**Header height is set in 1 place (for the links)** fix it according to this requirement